### DISCUSSION TITLE
Spotlights in Zoom

### DISCUSSION THREAD

**Discussion by danielengels** [sircharlo/meeting-media-manager#1172] [Q&A] [answered]
As I saw the release notes "A new OBS scene selector for making the display of Zoom participants easier (see https://github.com/sircharlo/meeting-media-manager/discussions/987 and https://github.com/sircharlo/meeting-media-manager/pull/1022 for details)"... the problem is the changing of the spotlight in zoom or not? 
You can control Zoom via Javascript, when you connect to the zoom meeting with their sdk... I do something to control for the meetings from the stage for the speaker. maybe take a look on  jwzoom.de and if you had question, you can write me...

**Comment by mtdvlpr** [comment, marked as answer]
@danielengels, the problem was to show Zoom participants in the kingdom hall with media to show. Previously you'd have to either switch spotlight for every image or have a Zoom obs scene and set a spotlight and pin where you'd need to continuously toggle the media window. The second workflow has now been optimized with the Zoom scene selector which will automatically toggle the media window when sensible.

That being said, in an ideal world I would integrate Zoom similarly to OBS in M3 where we could control mic/camera/spotlight of the host by sending a message to Zoom from our javascript code. Are you telling me that's already possible? I've searched all over Zoom's documentation but couldn't find what I was looking for.

**Reply by danielengels** [reply]
Yeah, please go to https://jwzoom.de and you can try. It is in German but i don't think it is too hard to understand. You have to give this zoom user the co host rights and then you can control the waiting room, some zoom settings, raised hands, scenes with spotlights and mute/unmute, or auto renames or counting the zoom participants... To do, for some functions i directly use the zoom websocket but it is working pretty well

**Reply by mtdvlpr** [reply]
@danielengels, awesome work! I tested it, and although I didn't understand everything, it looks very promising. Would you be willing to help us implement something similar in M³?

**Reply by danielengels** [reply]
Sure I can try. I am not a JS and nuxt/vue expert, but i can try to do ;)

**Reply by mtdvlpr** [reply]
Awesome! I made an issue for it at #1186

**Comment by Chipperchap** [comment]


**Reply by mtdvlpr** [reply]
@Chipperchap, thanks for the video! Small comment: because of the background music, it's difficult to hear your voice/explanation.

**Reply by Chipperchap** [reply]
I will have another go 😁

By the way - what video editing software do you use? I am struggling to find one that is easy to use but has enough features to be useful. I tried one that looked promising (Shotcut) but didn't even have audio amplification/reduction! ...so I am still trialling different ones.

**Reply by mtdvlpr** [reply]
I only use VidCutter for ubuntu occasionally when I need to cut the JW stream meeting to only have the public talk.

**Reply by sircharlo** [reply]
VideoPad!
